scripts/src/main/resources/scripts/command/vscode

```diff
@@ -122,10 +122,55 @@ function doRunVsCode() {
 
 function doStartVsCode() {
   doConfigureVsCode -u
+  doAddRecmdt
```

Comment:
why are we calling this on every run rather than in `doSetup` analogue to the plugins?

Reply:
Done.